lib/test/TestParser.js

```diff
@@ -220,13 +224,48 @@ module.exports = class TestParser {
         path = parts[0];
         operator = parts[1];
         value = parts.slice(2).join(" ");
-
+        const missingParsedCollection = value.split("-").map(x => x.trim()).filter(x => x);
+        if (missingParsedCollection.length > 1) {
+            value = missingParsedCollection;
+        }
+        
         const assertion = new Assertion(interaction, path, operator, value);
         assertion.lineNumber = Util.extractLine(assertionString);
         assertion.validate();
         return assertion;
     }
 
+    parseStringUtterAssertion(interaction, utterAssertionString) {
```

Comment:
We should not be cutting and pasting code like this. Instead, we should enhance the existing routine (parseStringAssertion).

We can pass a parameter to it that indicates whether it is a "singleLineInteraction" - if so, rather than looking for a path here:
https://github.com/bespoken/skill-testing-ml/blob/d09cf0fbe0fbadd5c272ebedd0d6df5a56b3322d/lib/test/TestParser.js#L223

We just hard-code the path to be prompt.

It is MUCH better to not repeat the code. This is comlex parsing logic, and making sure it works correctly is that much more difficult if we have it in two places.